src/sentry/static/sentry/app/views/releases/detail/organization/releaseOverview.jsx

```diff
@@ -0,0 +1,212 @@
+              endpoint={`/organizations/${orgId}/releases/${encodeURIComponent(
+                version
+              )}/resolved/`}
+              query={query}
+              pagination={false}
+              renderEmpty={() => (
+                <Panel>
+                  <PanelBody>
+                    <PanelItem key="none" justify="center">
+                      {t('No issues resolved')}
+                    </PanelItem>
+                  </PanelBody>
+                </Panel>
+              )}
+              showActions={false}
+              params={{orgId}}
+              className="m-b-2"
+            />
+            <h5>{t('New Issues in this Release')}</h5>
+            <IssueList
+              endpoint={`/organizations/${orgId}/issues/`}
+              query={{
+                ...query,
+                query: 'first-release:"' + version + '"',
+                limit: 5,
+              }}
+              statsPeriod="0"
+              pagination={false}
+              renderEmpty={() => (
+                <Panel>
+                  <PanelBody>
+                    <PanelItem justify="center">{t('No new issues')}</PanelItem>
+                  </PanelBody>
+                </Panel>
+              )}
+              showActions={false}
+              params={{orgId}}
+              className="m-b-2"
+              />**/}
+            {hasRepos && (
+              <div>
+                {Object.keys(filesByRepository).map((repository, i) => {
+                  return (
+                    <RepositoryFileSummary
+                      key={i}
+                      repository={repository}
+                      fileChangeSummary={filesByRepository[repository]}
+                    />
+                  );
+                })}
+              </div>
+            )}
+          </div>
+          <div className="col-sm-4">
+            {hasRepos ? (
+              <div>
+                {release.lastCommit && (
+                  <LastCommit commit={release.lastCommit} headerClass="nav-header" />
+                )}
+                <CommitAuthorStats orgId={orgId} version={version} />
+                <h6 className="nav-header m-b-1">{t('Projects Affected')}</h6>
+                <ul className="nav nav-stacked">
+                  {release.projects.length === 0
+                    ? this.renderEmpty()
+                    : release.projects.map(project => {
+                        return (
+                          <ReleaseProjectStatSparkline
+                            key={project.id}
+                            orgId={orgId}
+                            project={project}
+                            version={version}
+                          />
+                        );
+                      })}
+                </ul>
+              </div>
+            ) : (
+              <Well centered className="m-t-2">
+                <HeroIcon src="icon-commit" />
+                <h5>Releases are better with commit data!</h5>
```

Comment:
it does need a translation but I was going to tackle this later since the render block is almost a straight copy paste from project releases. It might be a bigger job if we want to fix up the styles to fit our new conventions as well.